services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -621,7 +623,8 @@ public long deleteMeter(final DatapathId dpid, final long meterId)
             long flowCookie = flowStatsEntry.getCookie().getValue();
             if (flowCookie != DROP_RULE_COOKIE
                     && flowCookie != VERIFICATION_BROADCAST_RULE_COOKIE
-                    && flowCookie != VERIFICATION_UNICAST_RULE_COOKIE) {
```

Comment:
We should add check method isDefaultRule(terrible term...).

PS You can check highest bit of cookie - if it is set - it is default(terrible term...) rule. Such kind of check will avoid modification of code that process default(terrible term...) all over the code.

(minor)